highlighters/main/main-highlighter.zsh

```diff
@@ -986,6 +987,21 @@ _zsh_highlight_main_highlighter_highlight_dollar_quote()
   REPLY=$i
 }
 
+# Highlight backtick subshells
+_zsh_highlight_main_highlighter_highlight_backtick()
+{
+  local arg1=$1 i q=\` style
+  i=$arg[(ib:arg1+1:)$q]
+
+  if [[ $arg[i] == '`' ]]; then
+    style=back-quoted-argument
+  else
+    style=back-quoted-argument-unclosed
+  fi
+  _zsh_highlight_main_add_region_highlight $(( start_pos + $1 - 1 )) $(( start_pos + i )) $style
```

Comment:
You could use arg1 here.  It's more robust to use variable names without leading dollars; compare `s=1+1; echo $((2*s*2)) $((2*$s*2))`.